MyStack/MyStack/MyStackBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyStack
+{
+    public abstract class MyStackBase : IMyStack<int>
```

Comment:
MyStackBase is generic too!
public abstract class MyStackBase\<T\> : IMyStack\<T\>